scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/dynamic/syntax/SCollectionSyntax.scala

```diff
@@ -52,8 +78,8 @@ final class DynamicBigQueryOps[T](private val self: SCollection[T]) extends AnyV
     formatFn: T => TableRow,
     writeDisposition: WriteDisposition,
     createDisposition: CreateDisposition,
-    successfulInsertsPropagation: Boolean = false,
-    extendedErrorInfo: Boolean = false
+    successfulInsertsPropagation: Boolean,
+    extendedErrorInfo: Boolean
```

Comment:
should we create overload ?

Reply:
I'm ambivalent to/slightly against providing every possible override combination, so did not do that here.

Reply:
I'm fine with that